frontend/styles/custom.css

```diff
@@ -801,19 +809,11 @@ md-radio-button.md-default-theme.md-checked .md-off, md-radio-button.md-checked
 
 .xs-icon-button md-icon {
     font-size: 11px;
-    margin: auto;
+    margin: -2.1em auto auto auto;
     height: 11px;
     min-height: 10px;
 }
 
-.xs-delete-icon-justify md-icon {
```

Comment:
`comment.html` line 18 and 79 uses this class.

Reply:
Done.